pyproject.toml

```diff
@@ -47,7 +47,6 @@ docs = [
     "sphinx-wagtail-theme>=6.0,<7.0",
     "sphinxcontrib-spelling>=8.0,<9.0",
     "sphinx_copybutton>=0.5",
-    "pygments-graphql==1.0.0",
```

Comment:
**Question:** Out of curiosity, why doesn't pygments-graphql work? I checked the current version of the docs on RTFD and indeed the queries are not being highlighted.

Reply:
Originally apart of this thread https://github.com/torchbox/wagtail-grapple/pull/366#discussion_r1327305076, I think it's because when it's being used for types, for example:

```
id: ID
limit: PositiveInt
offset: PositiveInt
``` 

It's not technically valid syntax, so results in odd rendering, although I'm wondering if we should remove all `code-block:: graphql` refs as I thought it worked for valid query syntax we make use of, for example:

```
query {
    pages {
        ...on BlogPage {
            the_custom_field
        }
    }
}

```

If not, it's probably best to remove it completely.

Reply:
Had a look at the source code for pygments-graphql and it should have worked for types.
The tl;dr it was something on my list to tweak and play with because it wasn't working. 
May as well drop it - I am all for fewer dependencies

Reply:
I think full queries aren't getting rendered properly either, e.g. the example at the end of this section: https://wagtail-grapple.readthedocs.io/en/latest/general-usage/model-types.html#module-0 https://github.com/torchbox/wagtail-grapple/blob/d6d0ab7d2baef8d0e56dbbbed1a656b7adc61db0/docs/general-usage/model-types.rst#L127-L135

Reply:
the whole thing doesn't work, hence dropping it :D